ccla-managers-and-ccla-signatories/whitelist-contributors.md

```diff
@@ -1,15 +1,15 @@
-# Whitelist Contributors
+# Approved List Contributors
 
-Whitelists are lists of domain names, email addresses of individuals, GitHub usernames, or GitHub organization names who are authorized to contribute under a signed Corporate CLA. As a CCLA manager, you allow contributions to projects on behalf of your company by using any whitelist:
+Whitelists are lists of domain names, email addresses of individuals, GitHub usernames, or GitHub organization names who are authorized to contribute under a signed Corporate CLA. As a CCLA manager, you allow contributions to projects on behalf of your company by using any Approved List:
```

Comment:
The Approved List are lists of domain names...

Reply:
Sorry but I don't get it what is the change?

Reply:
The first word in this line is whitelist,...you missed it

Reply:
Solved